src/components/counter/Counter.tsx

```diff
@@ -1,22 +1,36 @@
-import React, { useState } from 'react';
-import styled from 'styled-components';
-
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { isAMultipleOfFive } from '../../utils/numbers'
 
 const Counter = () => {
-  const [currentValue, setCurrentValue] = useState(0)
-  const incrementValue = () => setCurrentValue(currentValue + 1);
+  const [currentValue, setCurrentValue] = useState<number>(0)
+  const [showMessage, setShowMessage] = useState<boolean>(false)
+  const incrementValue = () => setCurrentValue(currentValue + 1)
+  useEffect(() => {
+    if (isAMultipleOfFive(currentValue)) return setShowMessage(true)
+    setShowMessage(false)
+  }, [currentValue])
```

Comment:
While this code does what it should (congrats on that!) I have a couple of suggestions:
1. It's a matter of personal preference but, can we agree that on this repo single line `ifs` are not to be used?
My reasoning is that:
```js
//this is not as clear
if (condition) someAction()
someMoreCode()

//as this
if (condition) {
  someAction()
}
moreCode()
```
In my experience, `if` statements are good candidates for unit testing and the more easily I can identify them, the better.

2. Early exits (using return instead of else), kindof reminds me of old `go to` statements. I think a better syntax would be:
```js
useEffect(() => {
  if (isMultipleOfFive(currentValue)) {
    setShowMessage(true)
  } else {
    setShowMessage(false)
  }
});
```
This sort of thing is to be agreed on each individual repo, but to me, the more explicit branching code is, the better.